spec/index.html

```diff
@@ -865,16 +867,17 @@ <h2>Annotation Syntax</h2>
       <p>Turtle also defines an <dfn data-lt="annotation-syntax">annotation syntax</dfn>
         to both reify and assert a <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">triple</a>,
         which provides a convenient shortcut.
-        An annotation can be used to both assert a triple,
+        An annotation can be used to simultaneously assert a triple,
         via an explicit or implicit identifier,
         and have that triple be the
         <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a> of further triples.
-        If explicitly identified, the same reifier can then be used as the 
+        If explicitly identified, the same reifier can then be used as the
+        <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a> or
         <a data-cite="RDF12-CONCEPTS#dfn-object">object</a> of additional
         triples and/or <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a>.
         As with a <a>reified triple</a>,
         the annotation syntax allows the definition of
-        an <a>reifier</a> as either an <a href="#grammar-production-IRIREF"><code>IRIREF</code></a>
+        an <a>reifier</a> as either an <a href="#grammar-production-iri"><code>iri</code></a>
```

Comment:
```suggestion
        a <a>reifier</a> as either an <a href="#grammar-production-iri">IRI</a>
```

Reply:
Fine with `a` instead of `an`, but needs to remain `iri`, not `IRI`.